okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -650,8 +650,6 @@ static Status toGrpcStatus(ErrorCode code) {
 
     @Override
     public void run() {
-      String threadName = Thread.currentThread().getName();
-      Thread.currentThread().setName("OkHttpClientTransport");
```

Comment:
I think this has been helpful to teams. Is there an easier way to disable it just when it is unavailable?